src/main/java/com/github/packageurl/PackageURL.java

```diff
@@ -369,11 +369,11 @@ private void validateKey(final String value) throws MalformedPackageURLException
         validateChars(value, PackageURL::isValidCharForKey, "qualifier key");
     }
 
-    private String validatePath(final String value, final boolean isSubpath) throws MalformedPackageURLException {
+    private String validatePath(final String value) throws MalformedPackageURLException {
```

Comment:
If this is only used for the `subpath`, maybe it could be called `validateSubpath`?

Reply:
Done.